_overviews/scala3-macros/tutorial/reflection.md

```diff
@@ -12,13 +12,23 @@ It provides a view on the *Typed Abstract Syntax Trees* and their properties suc
 ## How to use the API
 
 Accessing this API needs an import that depends the current `Quotes`.
+The following definition
```

Comment:
The text is somewhat fragmented here. Should it be something like "The following code example illustrates / gives the definition for / ..."?